codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ViewServiceSourceGenerator.scala

```diff
@@ -140,44 +142,45 @@ object ViewServiceSourceGenerator {
         |    new ${view.providerName}(viewFactory, viewId)
         |
         |  override final def serviceDescriptor: Descriptors.ServiceDescriptor =
-        |    ${view.fqn.parent.javaOuterClassname}.getDescriptor().findServiceByName("${view.fqn.protoName}")
+        |    ${view.fqn.parent.name}Proto.javaDescriptor.findServiceByName("${view.fqn.protoName}")
```

Comment:
This is still wrong, but let's tackle that in a separate PR.
With ScalaPB the descriptors can be found in `*Proto` classes. In the sample it should use `CustomerViewProto` where I think `CustomerView` is derived from the file name `customer_view.proto`.

The above code results in `CustomerByNameProto` instead of `CustomerViewProto`, i.e. view.fqn.parent.name is the rpc service name.